src/components/form/examples/form-layout.tsx

```diff
@@ -4,6 +4,49 @@ import { schema } from './schema-layout';
 
 /**
  * Layout
+ * By default, each item in a limel-form will be rendered in a single row; and
+ * each row occupies the entire available width of the form's container.
+ *
+ * This default layout may work fine on small screens or narrow containers,
+ * but on larger screens it won't usually produce a nice layout. Thus we
+ * recommend that you choose an appropriate responsive layout for your form.
+ *
+ * ###### Enabling responsive layouts
+ *
+ * By specifying a `grid` layout `type` in your `schema`, as well as your desired
+ * number of `columns`, you can leave the job of responsively handling the form
+ * layout to Lime Elements.
+ * ````ts
+ * export const schema = {
+ *     type: 'object',
+ *     lime: {
+ *         layout: {
+ *             type: 'grid',
+ *             columns: 3,
+ *         },
+ *     },
+ *     …
+ * };
+ * ````
+ * :::note
+ * Value for `columns` can only be `5`, `4`, `3`, `2`, or `1`. If you do not
+ * specify a value, `limel-form` will choose `5` by default.
+ * :::
+ *
+ * So if you have chosen `4` for instance, the form will do its best to fit in
+ * four columns in a row. But for smaller containers in which placement of four
+ * items per row is not possible, the form will automatically change the layout
+ * and fit in 3 items per row. As the container's width decreases, number of
+ * columns will also decrease.
+ *
+ * :::tip
+ * You can divide a form into collapsible or none-collapsible sections,
+ * and for each section specify a different layout.
+ * :::
+ *
+ * In this example, each collapsible section has its on `colSpan`.
```

Comment:
Thanks @adrianschmidt . I'll do a fixup.
********
I can just say i **DO HATE 🤬** Mackbooks' new freaking [butterfly keyboard](https://www.thenextweb.com/plugged/2019/07/07/what-hell-apple-butterfly-keyboard-keys-design/). It's disgusting. Some letters randomly get typed twice. And some never get typed. Now add this to mmy (⬅ look, as I'm typing) dyslexia. And it feels you're typing on some disgusting low quality toy-keyboard and I can't stand the sound of it neither. It's like someone is chewing with open mouth. 😭 Blä🤮. 

Seems many people have this problem when you google it.

Reply:
Yeah, I have the same shitty keyboard on mine. I have to take the keys off every now and then to fix stuck keys… (if you do, it's important to open them on the top side, as there are hooks on the bottom side that would break if you do it wrong) https://apple.stackexchange.com/questions/272038/macbook-pro-2016-keyboard-key-stuck-how-to-remove-key